frontend/.prettierrc

```diff
@@ -3,5 +3,6 @@
     "tabWidth": 2,
     "printWidth": 100,
     "singleQuote": true,
-    "trailingComma": "none"
+    "trailingComma": "none",
+    "endOfLine": "auto"
```

Comment:
all good if we leave this as `"lf"` (or remove it, since this is the default anyway) to ensure consistency?

Reply:
Should be kept as `"lf"` and then change own vscode settings to add a `lf` instead of `crlf`